mail-filter/sqlgrey/files/sqlgrey.service

```diff
@@ -3,7 +3,19 @@ Description=SQLgrey Postfix Grey-listing Policy service
 After=network.target
 
 [Service]
+User=sqlgrey
+Group=sqlgrey
 ExecStart=/usr/sbin/sqlgrey
+PermissionsStartOnly=true
+ExecStartPre=/usr/bin/touch /run/sqlgrey.pid && /bin/chown sqlgrey:sqlgrey /run/sqlgrey.pid
```

Comment:
I don't like this. Any reason not to use tmpfiles.d? Any reason to combine it with `PIDFile` below?